### PR TITLE
Portfolio: mobile version

### PR DESCRIPTION
**Pull request by daveblaq**
### Portfolio: mobile version
**On this webpage, I:**

- Added the about me section
- Added my projects section

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -92,3 +92,313 @@ nav {
...
+  display: flex;
+  justify-content: center;
+  margin-top: 64px;
+  margin-bottom: 22px;
+}
+
+.recent-work header {
+  display: grid;
+  justify-content: center;
+}
```
Comment (victorokoroji) [inline css/style.css:109]: - [x] I would strongly recommend you use `display: grid ` on the work section container and not on the inner element to layout the content of the work section. This gives you control to layout your content nicely and to make your website  responsive 😄.
e.g
```
recent-work  {
  display: grid;
  _other atttributes_
}
```

File: css/style.css [hunk 2]
```diff
@@ -92,3 +92,313 @@ nav {
...
+
+.card-area {
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  margin-left: 26px;
+  margin-right: 22px;
+  margin-top: 62px;
+}
```
Comment (victorokoroji) [inline css/style.css]: - [x] I think you should consider using `display: flex `to layout the content of each. This gives you control to layout your content nicely and also makes your UI responsive 📱 .

File: css/style.css [hunk 3]
```diff
@@ -92,3 +92,313 @@ nav {
...
+.lang {
+  background: #ebf0ee;
+  border: 1px solid #c1c7d0;
+  border-radius: 8px;
+  width: 327px;
+  height: 352px;
+  margin-top: 59px;
+  margin-left: 24px;
+  margin-right: 24px;
+}
```
Comment (victorokoroji) [inline css/style.css]: - [x] I would recommend you consider wrapping the technologies cards in a container and style using display: flex to layout the content of each card and make the section responsive 📱

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @daveblaq , 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- Good UI design ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 1 above)
Comment (daveblaq) [reply css/style.css:109]: Fixed

File: css/style.css (hunk 2 above)
Comment (daveblaq) [reply css/style.css]: I implemented this

File: css/style.css (hunk 3 above)
Comment (daveblaq) [reply css/style.css]: I worked on this

**Review by mukhammadrizooff — APPROVED** [review]
Hi @daveblaq, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No Linter errors ✔️ 
- Good Implementations ✔️
- Descriptive PR description and commit history ✔️  
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**